lang/rust/avro/src/schema.rs

```diff
@@ -206,42 +206,6 @@ impl From<&types::Value> for SchemaKind {
     }
 }
 
-// Implement `Display` for `SchemaKind`.
```

Comment:
Cool! Better to use the `Dipslay` trait. Who a `Schema::Ref` will be displayed?

Reply:
Try it!